README.adoc

```diff
@@ -215,6 +215,19 @@ list of available Rake tasks via:
 $ bundle exec rake -T
 ---------------------
 
+=== Code style
+
+As most other open-source projects vagrant-service-manager has a set of conventions
+about how to write code for it. It is much easier to understand a codebase when all the code in it is in
```

Comment:
I don't think this line "It is much easier to understand a codebase when all the code in it is in a consistent style." is necessary.

Without this line, the description looks simple and enough.